yarn-project/end-to-end/src/shared/cross_chain_test_harness.ts

```diff
@@ -67,22 +67,6 @@ export async function deployAndInitializeTokenAndBridgeContracts(
    */
   underlyingERC20: any;
 }> {
-  if (!underlyingERC20Address) {
-    underlyingERC20Address = await deployL1Contract(walletClient, publicClient, TestERC20Abi, TestERC20Bytecode, [
-      'Underlying',
-      'UND',
-      walletClient.account.address,
-    ]).then(({ address }) => address);
-  }
-  const underlyingERC20 = getContract({
-    address: underlyingERC20Address!.toString(),
-    abi: TestERC20Abi,
-    client: walletClient,
-  });
-
-  // allow anyone to mint
-  await underlyingERC20.write.setFreeForAll([true], {} as any);
```

Comment:
This just didn't work at all in cases where underlyingERC20Address was provided on the input because that was typically WETH or DAI address from forked mainnet and those contracts simply did not have `setFreeForAll` method. This change sneaked into master because the `uniswap_trade_on_l1_from_l2.test.ts` was never run in [this PR](https://github.com/AztecProtocol/aztec-packages/pull/10403) that introduced the change.